worker/categorize.py

```diff
@@ -403,11 +404,18 @@ def _is_viable_candidate(self, phrase):
                 return False
 
             titled_words = map(lambda x: x.istitle(), words)
-            if titled_words not in [[True, False, True], [True, True,True]]:
+            if titled_words not in [[True, False, True], [True, True, True]]:
                 return False
 
         return True
 
+    def _is_nested(self, keyword, other_keywords):
+        for other_keyword in other_keywords:
```

Comment:
Wouldn't this make it possible to accidentally remove both "Google" and "Google Maps"? Does this run for every keyword against the list of all other keywords (e.g. Will it run with both "Google" and "Google Maps" as the keyword?).